.babelrc

```diff
@@ -4,7 +4,8 @@
         [
             "@babel/preset-stage-0",
             {
-                "decoratorsLegacy": true
+                "decoratorsLegacy": true,
+                "pipelineProposal": "minimal"
```

Comment:
Why do you need to add pipelineProposal ? I wouldn't go for stage-0 implementations unless we really really need to for some reason.